lib/faraday/connection.rb

```diff
@@ -416,8 +414,6 @@ def url_prefix=(url, encoder = nil)
         basic_auth user, password
         uri.user = uri.password = nil
       end
-
-      uri
```

Comment:
`Lint/Void` looks for property setters that explicitly return some value. I really hope someone isn't doing something like `foo = conn.url_prefix = "bar"`, because this would break that. And, it's probably not doing what the user thinks it's doing... :)